File: Data Structures & Algorithms/two-integer-sum/submission-2.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {number[]}
     */
    twoSum(nums, target) {
        let obj={}
        for(let i=0; i<nums.length; i++){
            let complement = target-nums[i]
            if(obj.hasOwnProperty(complement)){
                return [obj[complement], i]
            }
            obj[nums[i]] = i
        }
        return []
    }
}

File: Data Structures & Algorithms/missing-number/submission-2.java
class Solution {
    public int missingNumber(int[] nums) {
        // n/2*(a+l)
        int n = nums.length;
        int sum = (n * (1 + n))/2;
        System.out.println(sum);
        System.out.println(n);
        for (int i = 0; i < n; i++) {
            sum -= nums[i];
        }

        return sum;
    }
}